Sources/ArcGISToolkit/Components/Form/FormInputs/SingleLineTextEntry.swift

```diff
@@ -62,5 +63,11 @@ struct SingleLineTextEntry: View {
             input: input
         )
         .padding([.bottom], elementPadding)
+        .onAppear {
+            text = model.feature?.attributes[element.fieldName] as? String ?? ""
```

Comment:
Once we switch to the core implementation, this would simply be `input.value`, as core will handle everything else.